src/commands/whoami.rs

```diff
@@ -116,7 +116,24 @@ pub(crate) fn fetch_accounts(user: &GlobalUser) -> Result<Vec<Account>> {
     let response = client.request(&account::ListAccounts { params: None });
     match response {
         Ok(res) => Ok(res.result),
-        Err(e) => anyhow::bail!(http::format_error(e, None)),
+        Err(e) => {
+            match e {
+                ApiFailure::Error(_, ref api_errors) => {
+                    let error = &api_errors.errors[0];
+                    if error.code == 9109 {
+                        // 9109 error code = Invalid access token
+                        StdOut::info("Token might be expired or might not have the necessary permissions. Please consider running `wranger logout` or re-authenticate wrangler through `wrangler login` or `wrangler config`.");
```

Comment:
```suggestion
                        StdOut::info("Your API token might be expired, or might not have the necessary permissions. Please re-authenticate wrangler by running `wrangler login` or `wrangler config`.");
```
Maybe we shouldn't even suggest logging out, if wrangler login will wipe out the previous token?

Reply:
Oops. Yes, wrangler login will completely override the previous token.